client/src/app/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form (ngSubmit)="onSubmit()" class="login-form">
+  <ion-item>
+    <ion-label position="stacked">E-mail</ion-label>
+    <ion-input
+      name="email"
+      type="email"
+      required
+      inputmode="email"
+      pattern="[a-zA-Z0-9\.]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+"
+      autocomplete="email username"
+      clearInput="true"
+      data-test="input-user"
+    ></ion-input>
+  </ion-item>
+  <ion-item>
+    <ion-label position="stacked">Password</ion-label>
+    <ion-input
+      name="password"
+      type="password"
+      required
+      autocomplete="current-password"
+      minlength="4"
+      clearInput="true"
+      clearOnEdit="false"
+      data-test="input-password"
+    ></ion-input>
+  </ion-item>
+  <ion-button
+    buttonType="button"
+    type="submit"
+    class="login-button"
+    expand="block"
+    data-test="login-button"
```

Comment:
```suggestion
```

If we don't use these attributes in the tests, we can leave them off.
Besides that, I think we shouldn't have "code only for the tests", as it then doesn't test "the real thing", but sometimes it might make sense....